File: Iniciante/URI 1037.hs
main :: IO ()
main = do
    valor' <- getLine
    let valor = read valor' :: Double

    if valor >= 0 && valor <= 25
        then putStrLn $ "Intervalo [0,25]"
    else if valor > 25 && valor <= 50
        then putStrLn $ "Intervalo (25,50]"
    else if valor > 50 && valor <= 75
        then putStrLn $ "Intervalo (50,75]"
    else if valor > 75 && valor <= 100
        then putStrLn $ "Intervalo (75,100]"
    else
        putStrLn $ "Fora de intervalo"

File: Iniciante/URI 1037.clj
(require '[clojure.string :as str])

(def valor (read-string (read-line)))

(cond
    (and (>= valor 0.0) (<= valor 25)) 
        (println "Intervalo [0,25]")
    
    (and (> valor 25) (<= valor 50)) 
        (println "Intervalo (25,50]")

    (and (> valor 50) (<= valor 75))
        (println "Intervalo (50,75]")

    (and (> valor 75) (<= valor 100))
        (println "Intervalo (75,100]")

    :else 
        (println "Fora de intervalo")
)